donkeycarManagerFront/src/js/utils.js

```diff
@@ -0,0 +1,38 @@
+// why are we using 2 wide identation again ?
```

Comment:
Linter with rules should be added

Reply:
There already is a linter, I just hate 2 wide indentation